### PR TITLE
[stable/concourse] Add "worker.podManagementPolicy"

### PR DESCRIPTION
**Pull request by matthope**

### CODE REVIEW

**Comment by matthope** [comment]
Currently, I've set the default podManagementPolicy to "OrderedReady", which matches the previous behavior.

Its probably saner and more appropriate to default this to "Parallel".

**Comment by matthope** [comment]
/assign @mattfarina
(per @k8s-ci-robot )
cheers :D

File: stable/concourse/templates/worker-statefulset.yaml [hunk 1]
```diff
@@ -163,3 +163,6 @@ spec:
   updateStrategy:
     type: {{ .Values.worker.updateStrategy }}
 {{- end }}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "7") }}
+  podManagementPolicy: {{ default "Parallel" .Values.worker.podManagementPolicy }}
```
Comment (unguiculus) [inline stable/concourse/templates/worker-statefulset.yaml]: `default "Parallel"` is unnecessary.

File: stable/concourse/templates/worker-statefulset.yaml [hunk 2]
```diff
@@ -163,3 +163,6 @@ spec:
   updateStrategy:
     type: {{ .Values.worker.updateStrategy }}
 {{- end }}
+{{- if and (eq .Capabilities.KubeVersion.Major "1") (ge .Capabilities.KubeVersion.Minor "7") }}
```
Comment (unguiculus) [inline stable/concourse/templates/worker-statefulset.yaml]: We officiall support the current and previous minor versions, which would be 1.7+. The if would not be required.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/concourse/templates/worker-statefulset.yaml (hunk 1 above)
Comment (matthope) [reply stable/concourse/templates/worker-statefulset.yaml]: Good point, will fix.

File: stable/concourse/templates/worker-statefulset.yaml (hunk 2 above)
Comment (matthope) [reply stable/concourse/templates/worker-statefulset.yaml]: Cheers, Updated.

**Pull request merged**